### ISSUE TITLE
[BUG] bg-transparent reported as custom by no-custom-classname

### ISSUE THREAD

**Issue by bradleyayers** [francoismassart/eslint-plugin-tailwindcss#404]
**Describe the bug**

When I use `bg-transparent` it's reported as an error:

> Classname 'bg-transparent' is not a Tailwind CSS class [tailwind/no-custom-classname](https://github.com/francoismassart/eslint-plugin-tailwindcss/tree/master/docs/rules/no-custom-classname.md)

```
<div className="bg-transparent" />
```

**Expected behavior**

As per tailwind 3.x and 4.x bg-transparent is supported.

**Screenshots**

<img width="672" alt="Image" src="https://github.com/user-attachments/assets/65b7e3df-7ac6-4f7e-992f-266d8acfe2b3" />

**Additional context**

I've worked around the issue by adding `{ whitelist: ["bg-transparent"] }`.

I wonder if it's related to #271 in that `transparent` isn't recognised as a supported color.

**Comment by bradleyayers** [comment]
This turned out to be because I was overriding the color palette and didn't define `transparent`. I defined `theme.colors` rather than `theme.extend.colors`, and this removed the default `transparent` color.
